tophat/jenkins-timeline-plugin
# Remove yarn + fix `clean_webapp`

## Description
This replaces `yarn` with `npm` to install the app, removing the reliance on a globally installed `yarn`. It also fixes `make clean_webapp` by using a 'safer' operation that doesn't generate make-blocking errors.

## DevQA
### DevQA Prep
### DevQA Steps
- Run `make build_webapp` and verify that the old build artifacts are purged before the new ones are built.
- Run `make clean_webapp` and verify that it deletes everything in `src/main/webapp` except `jenkins_assets/*`, which contains non-webapp contents used by the Java portion of the plugin.

### Comment:
